app/helpers/feeder_form_builder.rb

```diff
@@ -113,6 +114,11 @@ def search_select(method, choices, params, html_options = {})
     select(method, choices, {include_blank: true, selected: selected}, add_search_action(html_options))
   end
 
+  def toggle_field_select(method, choices, params, html_options = {})
```

Comment:
in hindsight, I'm not sure if a new kind of select was totally necessary to create. i can't remember if/why i had issues adding this toggle action to a normal `select` field.